frontend/components/details/ModuleLink.tsx

```diff
@@ -0,0 +1,67 @@
+import ModuleModal from "@/components/details/ModuleModal";
+import { ModuleUnion } from "@/types/modules";
+import { Box } from "@material-ui/core";
+import axios from "axios";
+import { FC, MouseEventHandler, ReactElement, useState } from "react";
+
+const allowedModels = new Set([
+  "propositions",
+  "entities",
+  "images",
+  "quotes",
+  "sources",
+  "topics",
+]);
```

Comment:
These will be singular eventually, but for now they match current module link hrefs.

Reply:
Should we add a comment?